src/ShortScheduleCommand.php

```diff
@@ -50,9 +51,29 @@ public function isRunning(): bool
     }
 
     public function run(): void
+    {
+        $this->pendingShortScheduleCommand->getOnOneServer() ? $this->processOnOneServer() : $this->processCommand() ;
+    }
+
+    private function processOnOneServer()
+    {
+        if (Cache::missing($this->pendingShortScheduleCommand->cacheName())) {
+            Cache::add($this->pendingShortScheduleCommand->cacheName(), true, 60);
+
+            $this->processCommand();
+
+            while ($this->process->isRunning()) {
```

Comment:
Put this while look in a function called `waitForProcessToFinish` so we can lose that comment.